cmake/modules/SwiftSource.cmake

```diff
@@ -257,9 +257,9 @@ function(_compile_swift_files
                             "-Xfrontend" "${GROUP_INFO_JSON_FILE}")
   endif()
 
-  # Force swift 3 compatibility mode for Standard Library.
+  # Force swift 4 compatibility mode for Standard Library.
   if (SWIFTFILE_IS_STDLIB)
-    list(APPEND swift_flags "-swift-version" "3")
+    list(APPEND swift_flags "-swift-version" "4")
```

Comment:
I think you can remove this part altogether because we already define command line flags for stdlib and overlays elsewhere, so in practice this will end up building the stdlib with "-swift-version 4 -swift-version 4".

Reply:
While this is icky, I'll leave it for now because I want to move it to 4.2 after this.